docs_user/modules/con_ceph-daemon-cardinality.adoc

```diff
@@ -7,15 +7,12 @@ colocated within the same node.
 ifeval::["{build}" != "upstream"]
 For more information, see link:https://access.redhat.com/articles/1548993[Red Hat Ceph Storage: Supported configurations].
```

Comment:
is this only for upstream? why?

Reply:
Good catch: this should also be available downstream

Reply:
I checked the downstream and upstream builds, and this is actually conditionalized to appear downstream only. (I know it looks like it's meant for upstream). It's a Red Hat KCS, so I think it should only appear downstream.